deploy/charts/sdp-prototype/templates/workflow-testdask-deploy.yml

```diff
@@ -30,9 +30,3 @@ spec:
         resources:
           {{- toYaml . | nindent 10 }}
         {{- end }}
-      dnsPolicy: ClusterFirst
-      restartPolicy: Always
```

Comment:
Do we have a mechanism for stopping a workflow deployment when we're done, or do we not care about clean-up enough right now, and we'll just go for this, which gives us some resilience to workflow failure?

Reply:
The workflow can delete the deployment, which will in turn lead to the Helm release to be deleted, which in turn removes the Kubernetes deployments and eventually pods.

If the workflow crashes without undoing the deployment, the responsibility for cleaning up deployments would normally be with the processing controller - but we don't have that yet.

Reply:
Got it. OK.